RadixWallet/Clients/ImportLegacyWalletClient/ImportLegacyWalletClient+Live.swift

```diff
@@ -166,13 +168,10 @@ extension ImportLegacyWalletClient: DependencyKey {
 func convert(
 	parsedOlympiaAccount raw: Olympia.Parsed.Account
 ) throws -> OlympiaAccountToMigrate {
-	let bech32Address = try deriveOlympiaAccountAddressFromPublicKey(publicKey: raw.publicKey.intoEngine(), olympiaNetwork: .mainnet).asStr()
+	let address = try Sargon.deriveOlympiaMainnetAccountAddressFromPublicKey(
```

Comment:
Sargon will add support for `LegacyOlympiaAccountAddress` and return it directly.